CODESAMPLESLIST.md

```diff
@@ -157,13 +158,10 @@ This Page is a complete list of oneAPI Code Samples, sorted by Alphabetical orde
 |[Tutorials](https://github.com/oneapi-src/oneAPI-samples/tree/master/Libraries/oneCCL/tutorials)|['CPU', 'GPU']|Intel® oneCCL Tutorials|
 |[Tutorials](https://github.com/oneapi-src/oneAPI-samples/tree/master/Libraries/oneDNN/tutorials)|['CPU', 'GPU']|Intel® oneDNN Tutorials|
 |[UP2 LEDS](https://github.com/oneapi-src/oneAPI-samples/tree/master/Tools/IoTConnectionTools/up2-leds)|['CPU']|This sample shows how to use the LED class and APIs of the Eclipse* MRAA library. It is intended to run on the UP Squared board, and will utilize the 4 built-in color LEDs located under the Ethernet ports. No additional hardware is required|
-|[Use Library](https://github.com/oneapi-src/oneAPI-samples/tree/master/DirectProgramming/DPC++FPGA/Tutorials/Tools/use_library)|['FPGA']|An Intel® FPGA Tutorial demonstrating how to create Intel® FPGA libraries and to incorporate them in a Intel® oneAPI Data Parallel C++ (DPC++) project|
```

Comment:
@mdbtucker-any thoughts on maybe a better name in the json?

Reply:
@mdbtucker could you check it?

Reply:
The change to the CRR name (to CRR Binomial Tree) in the JSON looks fine to me.